nixos/doc/manual/administration/cleaning-store.xml

```diff
@@ -14,14 +14,14 @@ collector</emphasis> to remove old, unreferenced packages.  This is
 easy:
 
 <screen>
-$ nix-collect-garbage
+# nix-collect-garbage
```

Comment:
I think that's because you're not running via nix daemon, meaning everything needs write access to `/nix/*`.
```
$ echo $NIX_REMOTE
daemon
```